qutebrowser/config/configtypes.py

```diff
@@ -2014,6 +2014,15 @@ def __init__(self, none_ok: bool = False) -> None:
             ('last', "At the end."))
 
 
+class LogLevel(String):
+
+    """Log level."""
```

Comment:
```suggestion
    """A logging level."""
```

or something like that so that the docs look consistent (this ends up as a part of [the settings docs](https://github.com/qutebrowser/qutebrowser/blob/master/doc/help/settings.asciidoc#setting-types))